client/src/App.js

```diff
@@ -1,9 +1,10 @@
 import { useState, useEffect } from 'react';
+import { Route, Switch, useLocation } from 'react-router-dom';
 import styled from 'styled-components';
```

Comment:
I like to use `macro` for styled-components, but that's just personal preference :)
```suggestion
import styled from 'styled-components/macro';
```